resources/lang/en/addon.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+return [
+    'name'          => 'Integer',
+    'description'   => 'Integer field type.'
```

Comment:
If you are running PHPStorm - setup your codestyling for PSR standards. I always batch format but that will help you out with code styling standards. Just a little extra space is all ^_^